contentcuration/contentcuration/static/js/edit_channel/queue/views.js

```diff
@@ -27,7 +27,7 @@ var Queue = BaseViews.BaseWorkspaceView.extend({
     $trs: MESSAGES,
 
 	initialize: function(options) {
-		_.bindAll(this, 'toggle_queue', 'open_queue', 'close_queue', 'add_to_clipboard');
+		_.bindAll(this, 'toggle_queue', 'open_queue', 'close_queue', 'drop_in_clipboard');
```

Comment:
I think there might be some operations that call add_to_clipboard, so might be good to make sure all copying functions still work (clipboard -> item options -> make a copy, tree edit -> item options -> copy to clipboard, tree edit -> check items -> copy, tree view only -> open item -> copy, etc.)

Reply:
I've tested all of these and they work :) but I'll add the `'add_to_clipboard'` back to this list just to be sure, thanks for considering!